qiskit_addon_obp/utils/operations.py

```diff
@@ -15,7 +15,40 @@
 
 from __future__ import annotations
 
+import numpy as np
 from qiskit.quantum_info import SparsePauliOp
+from qiskit_ibm_runtime.utils.noise_learner_result import PauliLindbladError
+
+
+def _expand_op_and_qargs(
+    op: SparsePauliOp, op_qargs: list[int], other_qargs: list[int]
+) -> tuple[SparsePauliOp, list[int], list[int]]:
+    """Expands an operator and its qargs to include a second list of qargs.
+
+    Args:
+        op: the operator to expand.
```

Comment:
I know this is a private function, but can we please use the same grammar as the rest of the module? (e.g. capitalize the first letter of arg descriptions).